Cargo.toml

```
@@ -19,6 +19,7 @@ discv5 = { version = "0.3.1", features = ["serde"] }
 eth2_ssz = "0.4.0"
 ethereum-types = "0.12.1"
 ethportal-api = { path = "ethportal-api" }
+jsonrpsee = {version="0.16.2", features = ["async-client", "client", "macros", "server"]}
```

```suggestion
jsonrpsee = "0.16.2"
```

The features aren't needed to compile/run the testsuite